main/glossary/README.md

```diff
@@ -87,51 +84,47 @@ or `BigInt(123)`.
 The Board is a shared, on-chain location where users can post a value and make it
 accessible to others. When a user posts a value, they receive a unique ID for the value.
 Others can get the value just by knowing the ID. You can make an ID known by any
-communication method; private email, a DM or other private message, a phone call/voicemail,
-an email blast to a mailing list or many individuals, listing it on a website, etc.
+communication method&mdash;a DM or email or other private message, a phone call/voicemail,
+an email blast to a mailing list, publishing it on a website, etc.
 
 ## Brand
-Identifies the kind of [issuer](#issuer), such as "quatloos", "moola", etc. Brands are one of the two elements that
+Identifies the kind of [issuer](#issuer), such as "[Quatloos](#quatloos)",
+"[Moola](#moola)", etc. Brands are one of the two elements that
 make up an [amount](#amounts).
 For more information, see the [ERTP Guide's Brand section](/ertp/guide/amounts.md)
 and the [ERTP API's Brand section](/ertp/api/brand.md).
 
 ## Bundle
 
 Before a contract can be installed on Zoe, its source code must be bundled. This is done by:
-```
+```js
 import bundleSource from '@endo/bundle-source';
 const atomicSwapBundle = await bundleSource(
     require.resolve('@agoric/zoe/src/contracts/atomicSwap'),
 );
 ```
 The installation operation returns
 an `installation`, which is an object with one method; `getBundle()`. You can access an installed contract's source
-code via `const { source } = await E(installation).getBundle();`.
-In many cases, the bundled source is a single reviewable string.
-In others, the bundle contains to base 64 encoded zip file that you can
-extract for review.
-```
-jq -r .endoZipBase64 bundle.json | base64 -d > bundle.zip
+code via `const { endoZipBase64 } = await E(installation).getBundle();`.
+In most cases, the bundle contains a base64-encoded zip file that you can
+extract for review:
+```sh
+echo "$endoZipBase64" | base64 -d > bundle.zip
 unzip bundle.zip
 ```
 
 
 ## Burn
 
-Destroy all digital assets in a payment. See [`issuer.burn(payment, optAmount)`](/ertp/api/issuer.md#issuer-burn-payment-optamount).
+Destroy all digital assets in a payment, for example as part of consuming it in an exchange. See [`issuer.burn(payment, optAmount)`](/ertp/api/issuer.md#issuer-burn-payment-optamount).
 
 ## Comparable
```

Comment:
```suggestion
## Key
```
The `Comparable` type is now just a deprecated name for `Key`. All mentions of `Comparable` should be changed to `Key`.